src/platform/backends/shared/linux/linux_process.cpp

```diff
@@ -60,8 +60,13 @@ mp::LinuxProcess::LinuxProcess(std::unique_ptr<mp::ProcessSpec>&& spec) : proces
 
     // TODO: multiline output produces poor formatting in logs, needs improving
     QObject::connect(&process, &QProcess::readyReadStandardError, [this]() {
-        mpl::log(process_spec->error_log_level(), qPrintable(process_spec->program()),
-                 qPrintable(process.readAllStandardError()));
+        // Using readAllStandardError() removes it from buffer for Process consumers, so peek() instead.
+        // This copies the implementation of QProcess::readAllStandardError() replacing the read with peek.
```

Comment:
Nice, I was bitten by this when working on the recent `dnsmasq` stuff.

Reply:
Yep, sorry about that.